konfera/models/event.py

```diff
@@ -4,33 +4,23 @@
 from konfera.models.abstract import FromToModel
 
 
-CONFERENCE = 'conference'
-MEETUP = 'meetup'
-
-EVENT_TYPE_CHOICES = (
-    (CONFERENCE, _('Conference')),
-    (MEETUP, _('Meetup')),
-)
-
-DRAFT = 'draft'
-PUBLISHED = 'published'
-PRIVATE = 'private'
-EXPIRED = 'expired'
-
-EVENT_STATUS_CHOICES = (
-    (DRAFT, _('Draft')),
-    (PUBLISHED, _('Published')),
-    (PRIVATE, _('Private')),
-    (EXPIRED, _('Expired')),
-)
-
-
 class EventManager(models.Manager):
     def published(self):
-        return self.get_queryset().filter(status=PUBLISHED)
+        return self.get_queryset().filter(status='published')
 
 
 class Event(FromToModel):
```

Comment:
add also constants inside the class:

```
CONFERENCE = 'conference'
MEETUP = 'meetup'

DRAFT = 'draft'
...
```

reason for this is that everywhere in the code we would refer to constant. We do it because you will not have a typo in string (it is easy to make such mistake), and second if we need to change the value of the constant we do it and one place and everywhere in the code it is changed. 

For example if we change 'draft' to 'drafted' you would have to update all filters, validation methods, everywhere you compared to string, but if you use cont you change it just here.